alpakka-brave/src/test/java/no/sysco/middleware/alpakka/brave/javadsl/examples/HelloWorld.java

```diff
@@ -0,0 +1,35 @@
+package no.sysco.middleware.alpakka.brave.javadsl.examples;
+
+public class HelloWorld {
```

Comment:
Should this example be commented out? If so, maybe you can get rid of it?

Reply:
We can remove it in the meantime, I will add some proper tests.